src/state/index.ts

```diff
@@ -18,6 +18,11 @@ import { routingApi } from './routing/slice'
 
 const PERSISTED_KEYS: string[] = ['user', 'transactions', 'lists']
 
+let preloadedState = undefined
+if (process.env.NODE_ENV === 'production' || process.env.NODE_ENV === 'development') {
```

Comment:
i think we want !== 'test' or something

Reply:
I removed this block and instead use `load.disableWarning` to disable the first time use warning for test runs.